snmp/README.md

```diff
@@ -236,46 +127,314 @@ instances:
-| `<INSTANCE_CONFIG>`    | ```{"ip_address":"%%host%%", "port":"161", "community_string":"<COMMUNITY_NAME>", "metrics":[{"MIB":"<MIB_NAME>","symbol":"<SYMBOL>"},{"OID":"<OID>","name":"<OID_NAME>"}]}``` |
-
-##### SNMP v3
-
-**Note**: See the [SNMP Library reference][4] for all configuration options.
-
-| Parameter              | Value                                                                                                                                                                                                                                                                                                                  |
-| ---------------------- | -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------      |
-| `<INTEGRATION_NAME>`   | `snmp`                                                                                                                                                                                                                                                                                                                 |
-| `<INIT_CONFIG>`        | `{"mibs_folder":"<PATH_TO_ADDITIONAL_MIBS>"}`                                                                                                                                                                                                                                                                          |
-| `<INSTANCE_CONFIG>`    | ```{"ip_address":"%%host%%", "port":"161", "snmp_version":"3", "user":"<USER_NAME>", "authKey":"<PASSWORD>", "privKey":"<PRIVACY_TYPE_KEY>", "authProtocol":"<AUTHENTICATION_PROTOCOL>", "privProtocol":"<PRIVACY_TYPE>", "metrics":[{"MIB":"<MIB_NAME>","symbol":"<SYMBOL>"},{"OID":"<OID>","name":"<OID_NAME>"}]}``` |
-
-
-### Custom metrics
-Metrics collected by the SNMP integration are considered [custom metrics][10], which impacts your [bill][11].
+If necessary, additional metrics can be defined in the instances. These metrics are collected in addition to those in the profile.
+
+#### Metric definition by profile
+
+Profiles can be used interchangeably, such that devices that share MIB dependencies can reuse the same profiles. For example, the Cisco c3850 profile can be used across many Cisco switches.
+
+**Note**: Profiles require a local version of the MIBs to be referenced.
+
+* [Generic router](#generic-router-profile)
+* [F5 Big IP](#f5-big-ip-profile)
+* [Dell iDRAC](#dell-idrac-profile)
+* [Cisco Nexus](#cisco-nexus-profile)
+* [Cisco c3850](#cisco-c3850-device-profile)
+* [Cisco Meraki](#cisco-meraki-profile)
+
+##### Generic router profile
+
+The [generic router profile][7] collects the following metrics.
+
+MIBs needed for local reference: IF-MIB, IP-MIB, TCP-MIB, UDP-MIB.
```

Comment:
To be confirmed by @therve, but thanks to #5397 I don't think users customers still need to have MIBs used by built-in profiles installed locally, do they?

Reply:
I believe you're correct-- I'll remove